lib/scss_lint/linter/disable_linter_reason.rb

```diff
@@ -0,0 +1,39 @@
+module SCSSLint
+  # Checks for "reason" comments above linter-disabling comments.
+  class Linter::DisableLinterReason < Linter
+    include LinterRegistry
+
+    def visit_comment(node)
+      return unless node.value.first.match(COMMAND_REGEX)
```

Comment:
This will also require comments for the `enable` counterpart, so for code like the following:

```scss
// scss-lint:disable ...
p {
}
// scss-lint:enable ...
```

...I would get two lints. Perhaps we should only report disabling without a reason?